pkgs/development/python-modules/beanhub-extract/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pytestCheckHook
+, poetry-core
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "beanhub-extract";
+  version = "0.0.7";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "LaunchPlatform";
+    repo = "beanhub-extract";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Wt8ZCyCaERNXEd0/Q89QWUW/wGFSHAP2RZLhnv5xkgY=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = [
+    pytz
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "beanhub_extract"
+  ];
+
+  meta = with lib; {
```

Comment:
https://github.com/NixOS/nixpkgs/issues/208242

Reply:
I agree with Sandro here, it's fine if it's limited to a small scope (here `meta`, but I also find it okay for `options`). 👎 on changing it IMO.

Reply:
I don't have strong opinion here. I merely presenting people with information